### PR TITLE
Base for prometheus provisionning

### PR DESCRIPTION
**Pull request by bersace**

### CODE REVIEW

File: ui/temboardui/cli/prometheus.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+            return
+        logger.info("Stopping Prometheus.")
+        self.proc.terminate()
+        self.proc = None
+        os.unlink(self.app.config.temboard.home + "/prometheus.pid")
+
+
+def provision_instance(app, instance):
+    logger.info("Provisionning %s.", instance)
+    d = app.config.temboard.home + "/prometheus/instances.d"
```
Comment (pgiraud) [inline ui/temboardui/cli/prometheus.py:135]: Can we move this closer to the line where it is actually used?

File: ui/temboardui/cli/prometheus.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        home=home,
+        now=datetime.utcnow(),
+        temboard_version=temboardui.__version__,
+    )
+    with open(home + "/prometheus.yml", "w") as fo:
+        os.chmod(fo.name, 0o600)
+        fo.write(conf)
+
+
+PROMETHEUS_CONFIG_TEMPLATE = """\
```
Comment (pgiraud) [inline ui/temboardui/cli/prometheus.py:194]: can this also be moved up?

**Comment by pirlgon** [comment]
Nothing in the changelog? maybe we could introduce what will be made later?

**Comment by bersace** [comment]
> Nothing in the changelog? maybe we could introduce what will be made later?

Exactement.

**Review by pirlgon — APPROVED** [review]

**Pull request merged**